### PR TITLE
Dev to master PR

### PR DESCRIPTION
**Pull request by Remi749**
### Your checklist for this pull request
- [X] Make sure you are requesting to **pull a feature/bugfix/issue branch** (right side).
- [X] Make sure you are making a pull request against the **dev branch** (left side). Also you should start *your branch* off *our dev branch*.

### Description

PR to get master up to dev before new minor release **2.7.0**.

### CODE REVIEW

**Comment by olemp** [comment]
`PR to get master up to dev before new patch release.`

I think we should do minor release @Remi749?

File: src/js/WebParts/DynamicPortfolio/DynamicPortfolioConfiguration/IDynamicPortfolioColumnConfig.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { IColumn } from 'office-ui-fabric-react/lib/DetailsList'
 
 interface IDynamicPortfolioColumnConfig extends IColumn {
     readOnly: boolean;
-    render: 'Date' | 'Note' | 'Currency' | 'Status' | 'URL' | 'Default';
+    render: 'Date' | 'Note' | 'CalculatedPercentage' | 'CalculatedFloat' | 'Currency' | 'Status' | 'URL' | 'Default';
```
Comment (olemp) [inline src/js/WebParts/DynamicPortfolio/DynamicPortfolioConfiguration/IDynamicPortfolioColumnConfig.ts:5]: CalculatedPercentage and CalculatedFloat is cool beans.

File: src/js/Util/index.ts [hunk 2]
```diff
@@ -199,6 +199,16 @@ export function setItemFieldValue(fieldName: string, item: SP.ListItem, fieldVal
         case 'User':
         case 'UserMulti':
         case 'Boolean':
+        case 'Float': {
```
Comment (olemp) [inline src/js/Util/index.ts:202]: > CalculatedPercentage and CalculatedFloat is cool beans.

But it should match here.

**Review by olemp — CHANGES_REQUESTED** [review]

File: src/js/Util/index.ts (hunk 2 above)
Comment (Remi749) [reply src/js/Util/index.ts:202]: wups, will fix.

Comment (Remi749) [reply src/js/Util/index.ts]: fixed

**Pull request merged**